inscriptions/ISic000521.xml

```diff
@@ -190,11 +195,31 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                 <div type="textpart" subtype="fragment" n="a">
+                        <ab>
+                            <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><orig>SII</orig><gap reason="lost" extent="unknown" unit="character"/>
+                            <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><orig>CURT</orig><gap reason="lost" extent="unknown" unit="character"/>
+                            <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/>Lilyb <gap reason="lost" extent="unknown" unit="character"/>
+                        </ab>
+                </div>
+                <div type="textpart" subtype="fragment" n="b">
+                        <ab>
+                            <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">Lil</supplied>ybita<gap reason="lost" extent="unknown" unit="character"/> <gap reason="lost" extent="unknown" unit="character"/>
+                            <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><orig>AM</orig> <g ref="#interpunct">·</g> <orig>EX</orig><gap reason="lost" extent="unknown" unit="character"/>
+                            <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/><orig>TI</orig> <g ref="#interpunct">·</g> <orig>PUBI</orig><gap reason="lost" extent="unknown" unit="character"/>
+                            <lb n="4"/><gap reason="lost" extent="unknown" unit="character"/><orig>IPY</orig><gap reason="lost" extent="unknown" unit="character"/>
+                        </ab>
+                </div>
+                <div type="textpart" subtype="fragment" n="c">
+                        <ab>
+                            <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><orig>SC</orig><gap reason="lost" extent="unknown" unit="character"/>
+                            <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><orig>IIIV</orig><gap reason="lost" extent="unknown" unit="character"/>
+                        </ab>
+                 </div>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp><app><note>Text: CIL</note></app></listApp>
```

Comment:
could you add a note in the commentary that it is unclear if these fragments belong to one or several inscriptions